packages/kit/src/core/dev/plugin.js

```diff
@@ -105,7 +105,9 @@ export async function create_plugin(config, cwd) {
 						}),
 						routes: manifest_data.routes.map((route) => {
 							if (route.type === 'page') {
+								/**@type import('types').SSRPage**/
```

Comment:
`@type` doesn't do anything with `return`. You would need to declare a new variable or make it inline (`return /** @type {...} */ (...)`) but I don't think there's much value in it

```suggestion
```